docker-compose.yml

```diff
@@ -15,3 +17,19 @@ services:
         order: start-first
       restart_policy:
         condition: any
+  postgres:
+        container_name: postgres
+        hostname: postgres
+        image: postgres:latest
+        environment:
+            - POSTGRES_USER=postgres
+            - POSTGRES_PASSWORD=postgres
+            - POSTGRES_DB=pear
+        networks:
+            - main
+        ports:
+            - "5432:5432"
+        restart: on-failure
```

Comment:
we're publicly posting our db credentials and giving public port access? This makes our db publicly exposed? Please keep our db cred's secret and take away port mapping

Reply:
these are for local development -- notice `docker-compose.yml` vs `docker-compose.server.yml`; we're bringing up a local container instance of postgres, so it doesn't matter what we do here